transparency/transparency.go

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) The go-boot authors. All Rights Reserved.
+//
+// Use of this source code is governed by the license
+// that can be found in the LICENSE file.
+
+package transparency
+
+import (
+	"fmt"
+	"io/fs"
+
+	"github.com/usbarmory/go-boot/uefi/x64"
+	"github.com/usbarmory/boot-transparency/engine/sigsum"
+	"github.com/usbarmory/boot-transparency/policy"
+	"github.com/usbarmory/boot-transparency/transparency"
+)
+
+// Boot transparency configuration.
```

Comment:
godoc comments should begin with the structure name, therefore `Config represents a boot-transparency configuration`.